### PR TITLE
Add Ivory Coast as a supported country

### LINKED ISSUE #16001: Add Ivory Coast as a supported country
When installing a new version of PrestaShop,
Côte d'Ivoire appears in the list of available countries.

But after installation, no information about this country is available automatically.

Currency, taxes, ... are added manually step by step.

Thank you to see to automatically integrate this information to PrestaShop.

### PR DESCRIPTION
**Pull request by SylNicky**
Information needed to add Côte d'Ivoire localization pack

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Information needed to add Côte d'Ivoire as a localization pack (units, tax rates, language, currency)
| Type?         | new feature
| Category?     | LO 
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #16001
| How to test?  | http://www.izf.net/content/taxes-locales-et-tva-en-c-te-divoire, https://fr.coinmill.com/EUR_XOF.html#EUR=1, https://fr.wikipedia.org/wiki/Franc_CFA_(UEMOA), https://fr.iban.com/currency-codes

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16173)

### CODE REVIEW

**Comment by Progi1984** [comment]
Poke @LouiseBonnard

File: localization/ci.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<localizationPack name="Côte d'Ivoire" version="1.0">
```
Comment (LouiseBonnard) [inline localization/ci.xml]:
```suggestion
<localizationPack name="Ivory Coast" version="1.0">
```

File: localization/ci.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		<currency name="Franc CFA" iso_code="XOF" iso_code_num="952" sign="FCFA" blank="1" conversion_rate="655.957" format="2" decimals="0" />
+	</currencies>
+	<languages>
+		<language iso_code="fr" />
+	</languages>
+	<taxes>
+		<tax id="1" name="TVA CI 18%" rate="18" />
+		<tax id="2" name="TVA CI 10%" rate="10" />
+		
+		<taxRulesGroup name="CI Taxe standard (18%)">
```
Comment (LouiseBonnard) [inline localization/ci.xml]:
```suggestion
		<taxRulesGroup name="CI Standard Rate (18%)">
```

File: localization/ci.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			<taxRule iso_code_country="ci" id_tax="1" />
+		</taxRulesGroup>
+
+		<taxRulesGroup name="CI Taxe réduite (10%)">
```
Comment (LouiseBonnard) [inline localization/ci.xml]:
```suggestion
		<taxRulesGroup name="CI Reduced Rate (10%)">
```

Comment (SylNicky) [reply localization/ci.xml]: Ok

File: localization/ci.xml (hunk 2 above)
Comment (SylNicky) [reply localization/ci.xml]: ok

File: localization/ci.xml (hunk 1 above)
Comment (SylNicky) [reply localization/ci.xml]: OK

Comment (LouiseBonnard) [reply localization/ci.xml]: @SylNicky, thanks for the comment! You can click on the 'Commit suggestion' buttons to take those three modifications into account. After that, the pull request will be ready. ;-)

Comment (SylNicky) [reply localization/ci.xml]: Done

Comment (LouiseBonnard) [reply localization/ci.xml]: Thank you very much!

**Review by LouiseBonnard — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**